### PR TITLE
reviewed utils package

### PR DESCRIPTION
**Pull request by ghost**
- mainly simplified generateJson method

### CODE REVIEW

File: src/main/scala/org/dia/algorithms/mcc/MainCompute.scala [hunk 1]
```diff
@@ -40,8 +38,8 @@ object MainCompute {
   val rowDim = 180
   val columnDim = 360
   val TextFile = "TestLinks"
-  // Class logger
-  val LOG: Logger = org.slf4j.LoggerFactory.getLogger(this.getClass)
+  // Class loggerger
```
Comment (chrismattmann) [inline src/main/scala/org/dia/algorithms/mcc/MainCompute.scala]: typo, right? can you fix?

File: src/main/scala/org/dia/algorithms/mcc/MainCompute.scala [hunk 2]
```diff
@@ -98,13 +95,13 @@ object MainCompute {
     println(nodes.size)
     println(totEdges)
     println("*****************")
-//    val json = ("nodes" -> jsonNodes) ~ ("edges" -> jsonEdges)
-//    FileUtils.writeToFile("./resources/graph.json", pretty(render(json)))
+    //    val json = ("nodes" -> jsonNodes) ~ ("edges" -> jsonEdges)
```
Comment (chrismattmann) [inline src/main/scala/org/dia/algorithms/mcc/MainCompute.scala]: if the code is commented we should remove it, right?

**Comment by chrismattmann** [comment]
just a couple of minor updates @legendOfZelda if you get a chance update the PR and I'll merge thanks!

File: src/main/scala/org/dia/algorithms/mcc/MainCompute.scala (hunk 1 above)
Comment (ghost) [inline src/main/scala/org/dia/algorithms/mcc/MainCompute.scala]: yep, fixing it

File: src/main/scala/org/dia/algorithms/mcc/MainCompute.scala (hunk 2 above)
Comment (ghost) [inline src/main/scala/org/dia/algorithms/mcc/MainCompute.scala]: yes, doing it and committing soon.

**Comment by ghost** [comment]
ok. im finishing up MainCompute and commit soon. then im done with org.dia.algorithms.mcc

**Comment by ghost** [comment]
lots of files have the same changes because the Main... classes in org.dia.algorithms.mcc were already very similar and even shared some code exactly. need to abstract out some pieces to get rid of this redundancy, see #21

**Comment by chrismattmann** [comment]
looks good, merging! @legendOfZelda

**Pull request merged**